endpoints.json

```diff
@@ -1,43 +1,163 @@
 {
-  "GET /api": {
-    "description": "serves up a json representation of all the available endpoints of the api"
-  },
-  "GET /api/topics": {
-    "description": "serves an array of all topics",
-    "queries": [],
-    "exampleResponse": {
-      "topics": [{ "slug": "football", "description": "Footie!" }]
-    }
-  },
-  "GET /api/articles": {
-    "description": "serves an array of all articles",
-    "queries": ["author", "topic", "sort_by", "order"],
-    "exampleResponse": {
-      "articles": [
-        {
-          "title": "Seafood substitutions are increasing",
-          "topic": "cooking",
-          "author": "weegembump",
-          "body": "Text from the article..",
-          "created_at": "2018-05-30T15:59:13.341Z",
-          "votes": 0,
-          "comment_count": 6
-        }
-      ]
-    }
-  },
-  "GET /api/articles/:article_id": {
-    "description": "this is used to access an object containing all of the information regarding the passed item",
-    "exampleResponse": 
-      {
-      "article_id": 3,
-      "title": "Eight pug gifs that remind me of mitch",
-      "topic": "mitch",
-      "author": "icellusedkars",
-      "body": "some gifs",
-      "created_at": "2020-11-03T09:12:00.000Z",
-      "votes": 0,
-      "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
-    }
-  }
+	"GET /api": {
+		"description": "serves up a json representation of all the available endpoints of the api"
+	},
+	"GET /api/topics": {
+		"description": "serves an array of all topics",
+		"queries": [],
+		"exampleResponse": {
+			"topics": [{ "slug": "football", "description": "Footie!" }]
+		}
+	},
+	"GET /api/articles": {
+		"description": "serves an array of article objects",
+		"queries": [],
+		"exampleResponse": {
+			"articles": [
+				{
+					"author": "icellusedkars",
+					"title": "Eight pug gifs that remind me of mitch",
```

Comment:
we only need to put an example here so we don't need the full array - the user just wants to see what the structure of the data looks like, they dont want to know all of our test data